src/components/Clock.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    time: new Date().toLocaleTimeString(),
+  }
+
+  componentDidMount() {
+    this.intervalID = setInterval(() => {
+      this.setState({
+        time: new Date().toLocaleTimeString(),
+      });
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.name !== this.props.name) {
+      // eslint-disable-next-line
+      console.log(
+        `The Clock was renamed from ${prevProps.name} to ${this.props.name}`,
+      );
+    }
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.intervalID);
+  }
+
+  render() {
+    return (
+      // eslint-disable-next-line
+      <> {console.log(this.state.time)}
+        {this.state.time}
+      </>
```

Comment:
```suggestion
      <span>{this.state.time}<span/>
```
Better to move `console.log` into the `setInterval` method.
also, fragment is not neccessary here. It would be better to use more semantic tags